ui/src/utils/mod.rs

```diff
@@ -25,7 +25,9 @@ pub fn get_available_themes() -> Vec<Theme> {
         if file.metadata().unwrap().is_file() {
             let theme = file.path().display().to_string();
 
-            let theme_str = theme.split('/').last().unwrap();
+            let mut theme_str = theme.replace("\\", "/");
+            theme_str = theme_str.split('/').last().unwrap().to_string();
```

Comment:
Instead, you should be able to use `file.path().file_name()`  instead of splitting.  You can also iterate over the path and get the last item if you prefer that

eg

```rust
let theme_str = file.path().file_name().map(|s| s.to_string_lossy().to_string()).unwrap();
```

or 

```rust
let theme_str = file.path().iter().last().unwrap();
```

Note you do want to avoid unwrapping unless youre completely sure it wont panic.

Reply:
I liked the second one because it was clearer what it was returning . . . but I did have to make a few other changes since it was returning an osstr instead of just a str.